contracts/contracts/vault/VaultStorage.sol

```diff
@@ -88,8 +86,8 @@ contract VaultStorage is Initializable, Governable {
     // Address of the contract responsible for post rebase syncs with AMMs
     address private _deprecated_rebaseHooksAddr = address(0);
 
-    // Address of Uniswap
-    address public uniswapAddr = address(0);
+    // Deprecated: Address of Uniswap
+    address public _deprecated_uniswapAddr = address(0);
```

Comment:
We can change this to private if there is nothing use it. (We'll probably need to update frank's debugging info script at deploy time)

Reply:
Cool changing to private